NEO-TAT/tat_flutter
# Rework splash screen

## Description

This PR replaces splash screens across all platforms. NPC branding has been added to the bottom of the screens. Details about the changes can be found in the commit messages.

## Implementation

- [x] Add `splash-android_12.png` and its dark variant.
- [x] Add `branding.png` and its dark variant.
- [x] Remove unused images from the assets directory.

## Testing Instructions

Launch the app on each platform in both light mode and dark mode to check that the images are correctly displayed.

- [x] For Android 11 and older: [light](https://github.com/NEO-TAT/tat_flutter/assets/53939195/b45cd51f-dee8-4a98-9f2b-cedd36d03388), [dark](https://github.com/NEO-TAT/tat_flutter/assets/53939195/b2474967-59e5-4b79-8c7f-e87863ed4d71)
- [x] For Android 12 and later: [light](https://github.com/NEO-TAT/tat_flutter/assets/53939195/e4449350-0c46-41c2-a3c8-62e58c320274), [dark](https://github.com/NEO-TAT/tat_flutter/assets/53939195/41024385-5e58-4fd3-bec2-4bef096ceaef)

## Additional Notes

1. Due to the requirements of the new `SplashScreen` API in Android 12, the text "我的北科生活 TAT" has been removed from the splash screen.
2. Once [this upstream issue](https://github.com/jonbhanson/flutter_native_splash/issues/475) is resolved, we will be able to add [splash screen animations](https://developer.android.com/develop/ui/views/launch/splash-screen#launch).
3. For easier review, consider reviewing each commit separately and skipping the generated files.
